src/pathfinder/pathfinder_type.h

```diff
@@ -40,6 +40,9 @@ static const int YAPF_TILE_CORNER_LENGTH = 71;
  */
 static const int YAPF_INFINITE_PENALTY = 1000 * YAPF_TILE_LENGTH;
 
+/** Maximum length of ship path cache */
+static const int YAPF_SHIP_PATH_CACHE_LENGTH = 32;
```

Comment:
Do you have in mind to extend path cache to NPF? Then I would suggest renaming this const.
By the way, would be any problem if it is increased?

Reply:
Extending it to NPF isn't a bad idea, once this PR is merged.